3.distributed-calculator/deploy/node-divider.yaml

```diff
@@ -17,6 +17,7 @@ spec:
         dapr.io/enabled: "true"
         dapr.io/id: "divideapp"
         dapr.io/port: "4000"
+        dapr.io/config: "tracing"
```

Comment:
Should be called "appconfig". See my other comment further down for reasons why.

Reply:
Done in latest commit https://github.com/dapr/samples/pull/183/commits/670cd71d452c98433367297fa5e6948cb6ac6c6f. Thank you!